src/ontology/mondo-edit.obo

```diff
@@ -200289,7 +200290,7 @@ relationship: excluded_subClassOf MONDO:0015509 {source="Orphanet:586", source="
 relationship: excluded_subClassOf MONDO:0018396 {source="Orphanet:586", source="https://orcid.org/0000-0001-5208-3432"} ! obsolete rare male fertility disorder with obstructive azoospermia
 relationship: excluded_subClassOf MONDO:0018409 {source="Orphanet:586", source="https://orcid.org/0000-0001-5208-3432"} ! obsolete rare genetic disorder with obstructive azoospermia
 relationship: has_characteristic HP:0000007 {source="MONDO:HPOA", source="OMIM:219700", source="Orphanet:586"} ! Autosomal recessive inheritance
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/1884 {source="OMIM:219700", source="PMID:20301428", source="https://orcid.org/0000-0001-9310-0163", source="https://orcid.org/0000-0002-4142-7153"} ! CFTR
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/1884 {source="PMID:20301428", source="https://orcid.org/0000-0001-9310-0163", source="https://orcid.org/0000-0002-4142-7153"} ! CFTR
```

Comment:
Newly added protected gene association https://github.com/monarch-initiative/omim/blob/main/data/protected-disease-gene.tsv#L16

Reply:
@twhetzel this OMIM source was removed, but it should have been kept since it is protected. 
I can update it manually, but the removal of the OMIM x-ref is incorrect.